terraform/iam.tf

```diff
@@ -27,17 +27,40 @@ EOF
 
 data "aws_iam_policy_document" "organisation-management" {
   statement {
-    sid    = "DenyAllApartFromOrganisationManagement"
-    effect = "Deny"
-    not_actions = [
-      # Note that this doesn't allow the account to delete Organisational Units
+    sid    = "AllowOrganisationManagement"
+    effect = "Allow"
+    actions = [
+      # Note that this doesn't destructive permissions for AWS Organizations
```

Comment:
This doesn't _what_ destructive permissions? 🤔 (I think you missed the word ‘grant’?)